ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.8.yaml

```
@@ -0,0 +1,793 @@
+    tag: assisted-installer-controller
+  assisted-test-infra:
+    name: ocm-2.8
+    namespace: edge-infrastructure
+    tag: assisted-test-infra
+  assisted-test-infra-internal:
+    name: ocm-2.8
+    namespace: edge-infrastructure
+    tag: assisted-test-infra-internal
+  cli:
+    name: "4.11"
+    namespace: ocp
+    tag: cli
+  cluster-api-provider-agent:
+    name: ocm-2.8
+    namespace: edge-infrastructure
+    tag: cluster-api-provider-agent
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  edge_infrastructure_swagger_codegen_cli:
+    name: edge-infrastructure-swagger-codegen-cli
+    namespace: ci
+    tag: 2.4.18
+  hypershift-operator:
+    name: "4.12"
+    namespace: ocp
+    tag: hypershift-operator
+  openshift_release_golang-1.18:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+  postgresql-12-centos7:
+    name: postgresql-12-centos7
+    namespace: edge-infrastructure
+    tag: latest
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: ci-images/Dockerfile.base
+  inputs:
+    openshift_release_golang-1.18:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.18
+  to: assisted-service-base
+- dockerfile_path: ci-images/Dockerfile.unit-tests
+  from: assisted-service-base
+  to: assisted-service-unit-test
+- dockerfile_path: ci-images/Dockerfile.lint
+  from: assisted-service-base
+  to: assisted-service-lint
+- dockerfile_path: ci-images/Dockerfile.subsystem
+  from: assisted-service-base
+  inputs:
+    cli:
+      as:
+      - quay.io/openshift/origin-cli:latest
+  to: assisted-service-subsystem
+- dockerfile_path: ci-images/Dockerfile.code-generation
+  from: assisted-service-base
+  to: assisted-service-generate
+- dockerfile_path: Dockerfile.assisted-service-build
+  inputs:
+    edge_infrastructure_swagger_codegen_cli:
+      as:
+      - quay.io/edge-infrastructure/swagger-codegen-cli:2.4.18
+    openshift_release_golang-1.18:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.18
+  to: assisted-service-build-base
+- dockerfile_literal: |
+    FROM base-build
+    COPY . .
+  inputs:
+    assisted-service-build-base:
+      as:
+      - base-build
+    src:
+      paths:
+      - destination_dir: ./
+        source_path: /go/src/github.com/openshift/assisted-service/.
+  to: assisted-service-build
+- dockerfile_path: Dockerfile.assisted-service
+  inputs:
+    openshift_release_golang-1.18:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.18
+  to: assisted-service
+- dockerfile_literal: |
+    FROM coreos/centos
+    COPY . .
+  from: src
+  to: assisted-service-scripts
+- dockerfile_literal: |
+    FROM coreos/centos
+    RUN yum install -y skopeo
+  from: src
+  to: assisted-service-publish-index
+operator:
+  bundles:
+  - dockerfile_path: deploy/olm-catalog/bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/edge-infrastructure/assisted-service:latest
```

don't we usually do here 
```suggestion
  - pullspec: quay.io/edge-infrastructure/assisted-service:ocm-2.8
```
?

I guess it makes sense if we also change it in the service repo (and I remember our flow usually includes it)

---

:eagle: